code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -629,7 +629,7 @@
 	if(iscarbon(exposed_mob) && exposed_mob.stat != DEAD)
 		var/mob/living/carbon/exposed_carbon = exposed_mob
 		var/holycheck = ishumanbasic(exposed_carbon)
-		if(reac_volume < 5 || !(holycheck || islizard(exposed_carbon) || (ismoth(exposed_carbon) && exposed_carbon.dna.features["moth_wings"] != "Burnt Off"))) // implying xenohumans are holy //as with all things,
+		if(reac_volume < 5 || !(holycheck || isflyperson(exposed_carbon) || isskeleton(exposed_carbon) || isandroid(exposed_carbon) || islizard(exposed_carbon) || (ismoth(exposed_carbon) && exposed_carbon.dna.features["moth_wings"] != "Burnt Off"))) // implying xenohumans are holy //as with all things,
```

Comment:
remove this from this pr, i'll make it atomized by removing this retarded check in the first place

Reply:
#59756

Reply:
sure